spec/requests/api/v1/course_spec.rb

```diff
@@ -552,5 +494,61 @@ def get_next_courses(link, params = {})
         expect(Subject.where(subject_code: "U3").count).to eq(0)
       end
     end
+
+    describe "age_range" do
+      context "when level is 'primary'" do
+        let(:course) { create(:course, provider: provider, age_range_in_years: '3_to_7', level: 'primary', subjects: [find_or_create(:primary_subject, :primary)]) }
```

Comment:
For the future: how about we add a `primary` trait to set the `subject`, `level` and `age_range_in_years`?

Reply:
Yep good idea. I do this in another PR